sdk/messaging/eventgrid/azsystemevents/models.go

```diff
@@ -1851,6 +1889,166 @@ type AppServicePlanEventTypeDetail struct {
 	Status *AsyncStatus
 }
 
+// AvsClusterCreatedEventData - Schema of the Data property of an CloudEvent/EventGridEvent for a Microsoft.AVS.ClusterCreated event.
+type AvsClusterCreatedEventData struct {
```

Comment:
Capitalize `Avs`?

Reply:
Ah, interesting. Yes, I can fix those. We'll have to get in the habit of reviewing these now, it's basically a monthly cadence at this point.

Reply:
Fixed!